qemu/tests/virtio_fs_daemon_readonly.py

```diff
@@ -0,0 +1,146 @@
+from avocado.utils import process
+from virttest import error_context, utils_disk, utils_misc, utils_test
+from virttest.utils_version import VersionInterval
+
+from provider import virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test virtio-fs daemon exported with --read-only option.
+    Steps:
+        1. Create a shared directory and a file for testing on the host.
+        2. Run the virtiofsd daemon with --readonly on the host.
+        3. Boot a guest on the host.
+        4. Log into guest then mount the virtiofs.
+        5. Test the mount dir is readonly.
```

Comment:
besides the writing test, do we need to check the access permission of the mounted dir in guest?

Reply:
Hi, What do you mean of access permission here? From my point of view that checked the read and write operations should be okay.